configuration/shuttle-service-name.mdx

```diff
@@ -0,0 +1,5 @@
+---
+title: "Shuttle Service Name"
+---
+
+The name of a Shuttle service (your project), can be set up front as part of the ```cargo shuttle init``` command. In the series of setup questions, the name is the first to be chosen.  Later, to change the name, a configuration file called ```Shuttle.toml``` can be created in the root of the project folder, at the same level as ```Cargo.toml```.  In this file, simply put: ```name = <your-project-name>```. As soon as the ```Shuttle.toml``` file exits, it will control the name of the Shuttle service and override other choices.
```

Comment:
For in-line code blocks like this you can use just a single backtick, three backticks are only needed for multi-line code blocks.

Reply:
Sounds good.  Will make the adjustments!